server/text_generation_server/habana_quantization_env.py

```diff
@@ -17,11 +18,27 @@
 
 def prepare_model_for_quantization(model):
     if is_quantization_enabled:
-        if os.getenv("USE_INC", "1") != "0":
-            from neural_compressor.torch.quantization import FP8Config, convert
-            config = FP8Config.from_json_file(quant_config)
-            model = convert(model, config)
-        else:
-            import habana_quantization_toolkit
-            habana_quantization_toolkit.prep_model(model)
+        htorch.core.hpu_set_env()
+        if model.config.model_type == "llama":
```

Comment:
Only llama model needs to do this? @schoi-habana , i remember Falcon did it too, right?

Reply:
yes OH does this for this series of models

https://github.com/huggingface/optimum-habana/blob/8d79a834c9571359745a491a686e702d28509ee1/examples/text-generation/utils.py#L424

Reply:
@mandy-li I haven't changed this code, just consolidated code into one place. 
Do we need to add all those list models here ? that will be a change from existing code.

Reply:
yes we should

Reply:
added